README.md

```diff
@@ -49,6 +49,100 @@ This will set up an Electrode webapplication which will have 2 of the above 6 mo
   - [Electrode Confippet](https://github.com/electrode-io/electrode-confippet)
   - [Electrode Javascript Bundle Viewer](https://github.com/electrode-io/electrify)
 
+## Progressive Web App (PWA) features supported by the Electrode framework  
+#### 1. Offline first  
+  Offline first lets your app run without a network connection. At the same time it provides a great performance boost for repeat visit to your web site.
+  This is done with a service worker and by pre-caching your static assets as well as run time caching of react routes.  
+  [Learn More](https://github.com/electrode-io/electrode-archetype-react-app/blob/master/README.md#2-cache)  
+#### 2. Add To Homescreen  
+  After visiting you website, users will get a prompt to add your application to their homescreen (web or mobile). Combined with offline caching, this means your web app can be used exactly like a native application.  
```

Comment:
The "Add to Homescreen" prompt heuristics are actually a little more complicated, and Chrome for Android will actually prompt the user to do it AFAIK. You can see them listed [here](https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android), but it's an ever-changing list of requirements.

We take care of most of those, but one that app authors should be aware of is:

> The user has visited your site at least twice, with at least five minutes between visits.

So they won't see the banner immediately on their first visit. Another thing to note is that users can still manually add to home from their mobile browser menu.